src/Common/src/Interop/User32/Interop.WS.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        /// <summary>
+        ///  Window styles
+        /// </summary>
+        public enum WS : uint
```

Comment:
```suggestion
        public enum WindowStyle : uint
```

Looking at the use, it probably should be decorated as `Flags`, and then name `WindowStyles`